projects/project-model.js

```diff
@@ -0,0 +1,12 @@
+
+
+module.exports = {
+	findName
+}
+
+function findName(db) {
+	return db('tasks as t')
+		.join('projects as p', 'p.Id', 't.projectId')
+		.select('p.name', 'p.description as ProjectDescription', 't.description as TaskDescription', 't.notes', 't.complete', 't.id ')
+		.orderBy('p.name')
+}
```

Comment:
Try as best as you can to separate all of your database queries into this file. This will help keep your routers clean and free of database queries so they can just focus on handling the `req` and `res` 👍